src/Tribe/Rewrite.php

```diff
@@ -559,12 +565,13 @@ protected function get_handled_rewrite_rules() {
 			$this->setup();
 		}
 
+		$all_rules     = isset( $this->rewrite->rules ) ? (array) $this->rewrite->rules : [];
```

Comment:
* :no_entry_sign: Spaces must be used to indent lines; tabs are not allowed
* :warning: Equals sign not aligned correctly; expected 1 space but found 5 spaces
* :no_entry_sign: A cast statement must not be followed by a space
* :warning: Line exceeds 80 characters; contains 93 characters

<sub>hash: 56937b35f0dfcc57c27bc5725eadf018</sub>